check/README.md

```diff
@@ -19,8 +19,8 @@ Renaming a key should be done with caution.
 Many of the keys in the stringtable are used in the game source code explicitly,
 or in content scripts.
 Some keys are also generated programatically, such as by adding a prefix to other text.
-C++ use `UserString` and `UserStringNop` to retrieve keys,
-Python `fo.userString`, and `fo.userStringList`.
+In C++ `UserString` and `UserStringNop` are used to get keys values,
```

Comment:
`UserStringNop` doesn't actually get a key's value... it just marks that a string is a key. It actually is a No-Op... does nothing and returns what it was passed, at the preprocessor stage.

Reply:
So basically `UserStringNop("KEY")` is a special syntax to write "KEY".  The only purpose of it is readability?

I use that same idea for the AI code for ids. https://github.com/freeorion/freeorion/blob/master/default/python/common/fo_typing.py

Reply:
The purpose is to be able to scan through the scource code and identify strings that are used that are expected to be present in the stringtable by the UI code. I don't know if anything actually does check this, but that's the idea.

Reply:
well as i see it, it informs human readers of the source code, that _**yes, someone thought about this, and decided that, no this is not for internationalization, so we put a mark on it, so noone has to think about that one again**_ or yes maybe some outomated testing in the future to make sure that all string literals are marked as either to be internationalized or marked as to be excluded from internationalization. the latter should really be very few, things like internal enum values.

Reply:
> yes, someone thought about this, and decided that, no this is not for internationalization, so we put a mark on it, so noone has to think about that one again

The opposite: Someone marked a string specifically as needing a user-readable stringtable entry. Ideally all strings that are written explicitly in code and that need a stringtable entry would be wrapped by `UserString` or `UserStringNop`.

If a string in C++ code doesn't need localization, then there is presently no way to mark it as such. And I don't think there should be, as there are many small internal-use string literals that don't need localization and marking them all would be highly intrusive and distracting throughout the code.